### PR TITLE
Migrate from Leap 15.4 to 15.5

### PR DESCRIPTION
**Pull request by mbussolotto**
## What does this PR change?

Migrate from Leap 15.4 to 15.5

## GUI diff

No difference.

- [ ] **DONE**

## Documentation
- TODO
- [ ] **DONE**

## Test coverage
- No tests

- [x] **DONE**

## Links

Fixes https://github.com/SUSE/spacewalk/issues/22507

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: susemanager/bin/server-migrator.sh [hunk 1]
```diff
@@ -56,8 +56,11 @@ if [[ "${CURRENT_VERSION_ID}" != "\"${NEW_VERSION_ID}\"" ]]; then
...
 
 echo "==================================================================="
-echo "OS migrated successfully, now please migrate to the new postgres version "
-echo "by running /usr/lib/susemanager/bin/pg-migrate-x-to-y.sh"
+echo "OS migrated successfully"
+
+# Upgrade from 15.4 to 15.5 will not migrate postgres, so the 2 lines below are commented.
+#echo "Now please migrate to the new postgres version "
+#echo "by running /usr/lib/susemanager/bin/pg-migrate-x-to-y.sh"
 echo
```
Comment (juliogonzalez) [inline susemanager/bin/server-migrator.sh:64]:
```suggestion
echo "IMPORTANT: Migrating postgres is NOT REQUIRED this time"
echo
```

**Review by juliogonzalez — COMMENTED** [review]
Maybe a message to make clear to the users that DB migration is not needed?

**Review by juliogonzalez — APPROVED** [review]
LGTM, but I'd like a second review from the release engineers :-)

**Review by raulillo82 — APPROVED** [review]
LGTM

**Pull request merged**